lib/browserify-transform.js

```diff
@@ -42,13 +36,16 @@ module.exports = function yoYoify (file, opts) {
     this.push(res)
     this.push(null)
   }
+
   function walk (node) {
+    var res
+
     if (isSupportedView(node)) {
       if (node.arguments[0].value === 'bel' ||
         node.arguments[0].value === 'choo/html' ||
         node.arguments[0].value === 'nanohtml') {
         // html and choo/html have no other exports that may be used
-        node.edit.update('{}')
+        node.edit.update('{ createElement: require("' + path.join(node.arguments[0].value, '/lib/createElement") }'))
```

Comment:
would it make sense to just do `update('require("/path/to/createElement")')` here and call that function directly as `html()` or `createElement()` (not renaming the variable is probably easier) instead of doing `html.createElement`?

`choo/html` is not going to have an export named `choo/html/lib/createElement`, and `bel` didn't have one either. maybe this should use `require.resolve('./createElement')` instead? or `path.relative()` from the input file

Reply:
Yeah, that was my first idea, but then I thought it wouldn't be a good idea to "reuse" an already existent api. I mean, one might want to:
```js
var html = require('nanohtml')
var foo = function (strings, ...keys) {
  // do stuff
  return html(strings, ...keys)
}
var bar = html`<span>HELLO</span>`
```

You are absolutely right about `bel` and `choo/html`. Regarding `choo/html`, I could still replace with `nanohtml/lib/createElement` or something like it. Regarding `bel`, I'm not so sure but probably it won't be supported in this version on `nanohtml` given that it doens't expose a `createElement` function